CMakeLists.txt

```diff
@@ -33,14 +33,51 @@ if(NOT "cxx_std_14" IN_LIST CMAKE_CXX_COMPILE_FEATURES OR NOT "cxx_digit_separat
       "This compiler does not fully support C++14, choose an higher version or an other compiler.")
 endif()
 
+# If CoreNEURON is built standalone then it has to know what version of NEURON to claim
+# compatibility with via NRN_VERSION_* macros in VERBATIM blocks.
+set(CORENRN_COMPATIBLE_NRN_VERSION_MAJOR 8)
+set(CORENRN_COMPATIBLE_NRN_VERSION_MINOR 2)
+set(CORENRN_COMPATIBLE_NRN_VERSION_PATCH 0)
```

Comment:
Just a note: I wondered if coreneuron could follow the same version numbers as neuron for simplicity. (but we want to get rid of coreneuron as standalone builds anyway...)

Reply:
I think that would be fine, it just seemed a [little] bit more invasive. If you think it's worth it then I'm happy to change?

Reply:
if you also think it's cleaner and simple, I am happy to make the change!

Reply:
👍, good to go!